packages/shared/src/config/modules.ts

```diff
@@ -74,7 +59,4 @@ export const moduleInfos: ModuleInfo[] = [
     puzzlesBanner,
     header,
     headerSupportAgain,
-    ausMomentHeaderNonSupporter,
-    ausMomentHeaderSupporter,
-    ausMomentBanner,
```

Comment:
shall we also delete the components?

Reply:
yeah that probably makes sense - I'll remove them all now